identity/identity.go

```diff
@@ -55,16 +61,17 @@ type System struct {
 
 // Identity is the main body of the XRHID
 type Identity struct {
-	AccountNumber         string    `json:"account_number,omitempty"`
-	EmployeeAccountNumber string    `json:"employee_account_number,omitempty"`
-	OrgID                 string    `json:"org_id"`
-	Internal              Internal  `json:"internal"`
-	User                  User      `json:"user,omitempty"`
-	System                System    `json:"system,omitempty"`
-	Associate             Associate `json:"associate,omitempty"`
-	X509                  X509      `json:"x509,omitempty"`
-	Type                  string    `json:"type"`
-	AuthType              string    `json:"auth_type"`
+	AccountNumber         string         `json:"account_number,omitempty"`
+	EmployeeAccountNumber string         `json:"employee_account_number,omitempty"`
+	OrgID                 string         `json:"org_id"`
+	Internal              Internal       `json:"internal"`
+	User                  User           `json:"user,omitempty"`
+	System                System         `json:"system,omitempty"`
+	Associate             Associate      `json:"associate,omitempty"`
+	X509                  X509           `json:"x509,omitempty"`
+	ServiceAccount        ServiceAccount `json:"service_account,omitempty"`
+	Type                  string         `json:"type"`
+	AuthType              string         `json:"auth_type"`
```

Comment:
Just to be clear, this field is now "omitempty".

Reply:
Picked up in the rebase, thanks!